### PR TITLE
BVAL-591 follow-up

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: sources/constraint-declaration-validation.asciidoc [hunk 1]
```diff
@@ -166,6 +165,22 @@ public class User {
...
+[source, JAVA]
+----
+public class User {
+    private Map<String, @Valid List<@Valid Address>> addressesByType;
+}
+----
+====
+
+In the last example, all `Address` objects contained in the lists of the `addressesByType` map will be validated.
+Two value extractor are invoked for this:
```
Comment (gsmet) [inline sources/constraint-declaration-validation.asciidoc]: extractors

File: sources/constraint-declaration-validation.asciidoc [hunk 2]
```diff
@@ -166,6 +165,22 @@ public class User {
 ----
 ====
 
+.Making a nested collection subject to graph validation
+====
+[source, JAVA]
+----
+public class User {
+    private Map<String, @Valid List<@Valid Address>> addressesByType;
```
Comment (gsmet) [inline sources/constraint-declaration-validation.asciidoc]: I don't think we need the `@Valid` on the List or do we?

Comment (gunnarmorling) [inline sources/constraint-declaration-validation.asciidoc]: That's the question essentially; I've filed [BVAL-601](https://hibernate.atlassian.net/browse/BVAL-601) to clarify it (last question in there). For now I'd prefer to stay safe and require them all along the path.

File: sources/constraint-declaration-validation.asciidoc (hunk 1 above)
Comment (gunnarmorling) [inline sources/constraint-declaration-validation.asciidoc]: Fixed and force-pushed.

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request closed without merging**